Dockerfile

```diff
@@ -2,6 +2,17 @@ FROM ubuntu:trusty
 MAINTAINER "Syncano DevOps Team" <devops@syncano.com>
 
 ENV LAST_REFRESHED 2016-03-25
+ENV SYNCANO_APIROOT https://api.syncano.io/
```

Comment:
this is common part except for Ruby, but I'm trying to contact lib current dev to make it common